frontend/public/locales/de/order.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
For the future, it's enough if you add the `t(translationKey)` in the component/views, you dont need to translate

Reply:
Okay